airbyte-integrations/connectors/source-xkcd/pyproject.toml

```diff
@@ -3,26 +3,25 @@ requires = [ "poetry-core>=1.0.0",]
 build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
-version = "0.1.10"
+version = "0.2.0"
 name = "source-xkcd"
-description = "Source implementation for Xkcd."
+description = "Source implementation for xkcd."
 authors = [ "Airbyte <contact@airbyte.io>",]
 license = "MIT"
 readme = "README.md"
 documentation = "https://docs.airbyte.com/integrations/sources/xkcd"
 homepage = "https://airbyte.com"
 repository = "https://github.com/airbytehq/airbyte"
-[[tool.poetry.packages]]
-include = "source_xkcd"
+packages = [ { include = "source_xkcd" }, {include = "main.py" } ]
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "0.83.0"
+airbyte-cdk = "^2"
```

Comment:
Current CDK version is 3.9 btw. But, delete the whole file, let's manifest-only this.